content/en/docs/components/pipelines/author-a-pipeline/components.md

```diff
@@ -230,11 +211,12 @@ def component1(text: str, output_gcs: Output[Dataset]):
 def component2(input_gcs: Input[Dataset]):
     return ContainerSpec(image='alpine', command=['cat'], args=[input_gcs.path])
 
-@pipeline(name='two-step-pipeline-containerized')
+@pipeline
 def two_step_pipeline_containerized(text: str):
     component_1 = component1(text)
     component_2 = component2(input_gcs=component_1.outputs['output_gcs'])
 ```
+In the above example, `component1` takes in a text and output it to a path as an artifact. Then, the `component2` retrieves the artifact output by the `component1` and prints it out.
```

Comment:
nit: WDYT about renaming these so that they are more descriptive of what they do? Maybe something like `create_dataset` and `print_dataset`?